cookbooks/cdo-varnish/libraries/helpers.rb

```diff
@@ -45,10 +45,12 @@ def normalize_paths(paths)
       (is_extension && path.match('/'))
       raise ArgumentError.new("Invalid path: #{path}")
     end
+    single_path = !path.match('\*')
```

Comment:
Got it, single_path really didn't say much to me.

The query regex seems harmless even if the path ends in a '*", because the query expression is optional.  Could you not simply say.

path << QUERY_REGEX

If not, something like this seems clearer:
path_ends_in_wildcard = path.match('*$')
...
path << QUERY_REGEX unless path_ends_in_wildcard